packages/material-ui/src/CheckboxWidget/CheckboxWidget.tsx

```diff
@@ -19,6 +22,11 @@ const CheckboxWidget = (props: WidgetProps) => {
     onFocus,
   } = props;
 
+  // Because an unchecked checkbox will cause html5 validation to fail, only add
+  // the "required" attribute if the field value must be "true", due to the
+  // "const" or "enum" keywords
+  const required = schemaRequiresTrueValue(schema);
```

Comment:
If HTML5 validation is turned off, should we still use `schemaRequiresTrueValue(schema)` or should we revert to `props.required`?

Reply:
That's a fair question and I don't know the answer honestly. This change in particular brings the behavior into alignment with the bootstrap theme (https://github.com/rjsf-team/react-jsonschema-form/blob/master/packages/core/src/components/widgets/CheckboxWidget.js#L51).  I'd defer to your direction on the question though, so just let me know if you'd like for me to make changes here.

Reply:
Ah, makes sense, let's just keep it the way you did it then.